src/main/kotlin/com/sourcegraph/cody/chat/CodyChatMessageHistory.kt

```diff
@@ -1,85 +0,0 @@
-package com.sourcegraph.cody.chat
-
-import com.intellij.openapi.project.Project
-import com.intellij.ui.components.JBTextArea
-import com.sourcegraph.cody.history.HistoryService
-import com.sourcegraph.cody.history.state.MessageState
-import java.util.*
-
-class CodyChatMessageHistory(
```

Comment:
removed in favor of [TextAreaHistoryManager.kt](https://github.com/sourcegraph/jetbrains/pull/1786/files#diff-7ee152a571f6e3a8ba8a5f2cf1d1e9642da2ed4b8c1164601fef2ba01a020e74) and [ChatPromptHistory.kt](https://github.com/sourcegraph/jetbrains/pull/1786/files#diff-2a9606f41be2726cbcb7152452526de41192dbdd371f85a8c8f5fa30aad16f58)